synapse/storage/databases/main/events.py

```diff
@@ -1955,6 +1955,7 @@ def store_event_search_txn(self, txn, event, key, value):
             key (str):
             value (str):
         """
+
```

Comment:
Since we didn't actually change this file, let's backout this change (it makes it clearer where the real changes are)!

```suggestion
```